go-controller/pkg/ovn/master.go

```diff
@@ -772,6 +772,26 @@ func (oc *Controller) deleteNode(nodeName string, nodeSubnet, joinSubnet *net.IP
 		return fmt.Errorf("Failed to clean up node %s gateway: (%v)", nodeName, err)
 	}
 
+	chassisName, stderr, err := util.RunOVNSbctl("--data=bare", "--no-heading",
+		"--columns=name", "find", "Chassis",
+		"hostname="+nodeName)
+	if err != nil {
+		klog.Errorf("Failed to get chassis name for logical switch %s: stderr: %q, error: %v",
+			nodeName, stderr, err)
+		return err
+	}
+
```

Comment:
also just do `return fmt.Errorf()` the caller will do the right thing of loggin it. Otherwise we are logging things twice